### PR TITLE
ARUHA-124: added appD utility

### PR DESCRIPTION
**Pull request by adyach**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -10,4 +10,5 @@ ADD scm-source.json /scm-source.json
 EXPOSE 8080
 
 # run the server when a container based on this image is being run
-ENTRYPOINT java $JAVA_OPTS -Dloader.path=/plugins -Djava.security.egd=file:/dev/./urandom -jar nakadi.jar
\ No newline at end of file
+ENTRYPOINT java $JAVA_OPTS $(appdynamics-agent) -Dloader.path=/plugins -Djava.security.egd=file:/dev/./urandom -jar \
```
Comment (v1ctor) [inline Dockerfile]: How does it work? Don't we need to have dependency?

Comment (valgog) [inline Dockerfile]: Oh, is there a way not to have appdinamics dependency like this in our open-source project?

Comment (adyach) [inline Dockerfile]: @v1ctor we use "zalando/openjdk" Docker base image that's why the image already has necessary env for that (at least they say that)

Comment (v1ctor) [inline Dockerfile]: I propose to add this to JavaOpts property in nakadi-private project.

Comment (adyach) [inline Dockerfile]: @valgog @v1ctor thanks! I will move it to private repo then.

**Pull request closed without merging**